bite-size-notes/Week-3-d-Optimal Control-Jiangwen Cheng.md

```diff
@@ -1 +1,130 @@
+# A Quick Note of Control Barrier Function 
+## Objectives
+1. To provide a foundational understanding of Control Barrier Functions
+2. To demonstrate the application of CBFs in real-world scenarios
+## Introduction 
+Control Barrier Functions (CBFs) is a critical component in the design of safety-critical control systems. As the demand for autonomous technologies and intelligent control systems continues to rise, ensuring the safety of these systems becomes paramount. CBFs play an indispensable role in this context by providing a robust mathematical framework to enforce safety constraints dynamically, keeping the system states within safe operational boundaries. In this note, we will discuss the fundamentals of CBFs from their theoretical basis to practical applications.  
```

Comment:
I actually like how you have introduced CBF's in a very concise way. Along with robustness to uncertainties, you can also add importance of CBF's w.r.t the stability and collision avoidance. You could also give a small example of a bike and a tree which Prof. Karen had mentioned in the class with a figure at the end (giving a visual reference).

Reply:
Actually, there are three figures used to illustrate the CBF with respect with the stability abd collision. Since there is a conflict when uploading the file. You could click three dots at the upper right corner to check the pdf version.